mylar/search.py

```diff
@@ -4147,12 +4147,12 @@ def searcher(
                             mylar_host = (
                                 '%s%s:%s%s'
                                 % (proto, mylar.CONFIG.HTTP_ROOT,
```

Comment:
This should be ``` % (proto, mylar.CONFIG.HTTP_HOST, mylar.CONFIG.HTTP_PORT, hroot)```

Reply:
Did not see that one.
Should be fixed now